scripts/package-distribution.ps1

```diff
@@ -68,7 +68,7 @@ foreach ($Build in $AllBuilds) {
     if ($Platform -eq "windows" -or $Platform -eq "uwp") {
         $Headers += Join-Path $HeaderDir  "msquic_winuser.h"
     } else {
-        $Headers += Join-Path $HeaderDir  "msquic_linux.h"
+        $Headers += Join-Path $HeaderDir  "msquic_posix.h"
```

Comment:
.lttng.so is also being grabbed below, and that won't work on macos.

Reply:
That already does a check if the binary exists or not, so that won't fail. But, the library name I think will. Let me check.